src/WebJobs.Script.WebHost/Program.cs

```diff
@@ -28,6 +33,10 @@ public static IWebHost BuildWebHost(string[] args)
         public static IWebHostBuilder CreateWebHostBuilder(string[] args = null)
         {
             return Microsoft.AspNetCore.WebHost.CreateDefaultBuilder(args)
+                .ConfigureServices(services =>
+                {
```

Comment:
This host sample is intended to be a super simple example of spinning up a script host for a non web host. There are some pretty cryptic configuration steps like this one below. Why are these needed and how would someone discover them?